packages/canon/src/components/Grid/Docs.mdx

```diff
@@ -22,11 +22,11 @@ import { spacingProperties } from '../../layout/sprinkles.css';
   The grid component is made of two parts: the grid container and the grid item. Import all parts and piece them together.
 </Text>
 <Source
-  code={`import * as Grid from "@backstage/canon/grid";
+  code={`import { Grid, GridItem } from "@backstage/canon/grid";
```

Comment:
How come? I do like `Grid.Item`. If you wanna keep that you can use `export Object.assign(Grid, { Item: GridItem })`

Reply:
Ah you're right. I'll change that.